src/integrations/watchers/indexable-term-watcher.php

```diff
@@ -43,6 +44,8 @@ class Indexable_Term_Watcher implements Integration_Interface {
 	protected $site;
 
 	/**
+	 * Gets the conditionals.
+	 *
 	 * @inheritDoc
 	 */
```

Comment:
Same question as before, should it be one of: (1)
```
	/**
	 * Returns the conditionals based in which this loadable should be active.
	 *
	 * @return array
	 */
```
or (2) 
```
	/**
	 * @inheritDoc
	 */
```
instead of the combination?